android/build.gradle

```diff
@@ -72,9 +72,13 @@ def resolveClientSideBuild() {
     return true
 }
 
+def isPackageBuild() {
+    return System.getenv("REANIMATED_PACKAGE_BUILD") == "1"
+}
+
 def isDeveloperMode() {
     // developer mode, to run Example app
-    return safeAppExtGet("isReanimatedExampleApp", false) || System.getenv("REANIMATED_PACKAGE_BUILD") == "1"
+    return safeAppExtGet("isReanimatedExampleApp", false) || isPackageBuild()
```

Comment:
Can we remove `|| isPackageBuild()` from here and use `isDeveloperMode() || isPackageBuild()` instead? What do you think?